### ISSUE TITLE
f14-b issues in next-branch (2019.2.0)

### ISSUE THREAD

**Issue by dirteat** [Zaretto/fg-aircraft#94]
Hi there,
just would like to mention a few issues reported on the forum with the next-branch, that remain unfixed since a while and render that fantastic plane quite hard to fly these days :-/

1) various buttons in the cockpits  are shifted compared to where they should be
https://forum.flightgear.org/viewtopic.php?f=4&t=36426
![image](https://user-images.githubusercontent.com/2045654/74690734-85f26180-51e0-11ea-82a7-9c9126a5ad36.png)

2) AP is unstable:  switch on the AP on the f-14b, I was close the qbarmax, but without the AP the plane is very stable.
I have made a short video, I am switching off the AP just before the last turn, you can see the plane stops wobbling around.
Am I doing something wrong? Is this expected?

https://forum.flightgear.org/viewtopic.php?f=4&t=36607#p357251

3) The joystick "ailerons" controls are actually controlling only the spoilers, the little "wing devices" on top of the wings. What is amazing is that the plane is flyable, but on all views, the ailerons stay perfectly horizontal [may that explains 2)?]

Cheers,
Chris.

**Comment by Zaretto** [comment]
1. this will get fixed.
2. I will investigate.
3. The F-14 doesn't have ailerons - what you are seeing are the wing spoilers being used to provide roll control; together with differential stabilator deflection this is the aircraft roll control.

**Comment by dirteat** [comment]
> The F-14 doesn't have ailerons

Ok, that explains. Thanks :)

**Comment by dirteat** [comment]
Hi Richard,
let me go on in this thread with some report on the 2020-flap-tunings branch.

I had a good flight tonight with the f14b, my first on multiplayer actually,  branch 2020-flap-tunings. Well, for carrier ops, DFCS made things much much easier, like you can fly the donut constantly (I was unable to do that properly before, now it is piece of cake, throttle in manual).

Some guy engaged me while doing my carrier ops :) So, I had an opportunity to fly the f14-b on the edge.




At strong G-turn, it seems that the 2020-flap-tunings branch makes the plane more unstable than the one currently on FG-addon. It is really hard to keep a constant bank angle for instance, the plane wants to grow an instability on the roll! I've tried deactivating DFCS while flying, but that did not change anything. Switching off SAS made the plane more stable however.

Another thing, which is actually the very concern of that issue here, the autopilot "attitude" CTRL+T is really unstable from Mach 0.7-0.8 and above on the pitch! What is really strange is that the "Altitude" one, CTRL+T followed by CTRL+A is actually very stable in the very same conditions. Hope it helps to fix that!

Edit:
I've just figured out that the file  .fgfs/aircraft-data/f-14b.xml is badly interfering with a lot of things. It was created by a previous version of the f-14b. Removing it and starting from scratch made most of the above-quoted problems gone! There is some oscillations on the roll at CTRL+T, that disappear with CTRL+A, but nothing as catastrophic as I was describing just above.

PS: It is not the first time that these aircraft-data/ files are seriously impairing aircrafts when a change of version occurs, maybe they should deserve some versioning as we do for the fgaddons.


Thanks again for that super nice plane, full of feelings!

**Comment by dirteat** [comment]
A bit more testing.

That roll instability at high speed is indeed related to SAS roll. With SAS roll ON, (DFCS or not), the plane gets unstable at high speed. Switching off only SAS roll solves essentially all problems at high speed.

But, low speed is the contrary. Without SAS roll engaged, the plane is dancing along the roll axis at low speed, but that's normal as far I remember. And with DFCS and SAS roll engaged, it is a super stable plane, but at low speed only.

**Comment by Zaretto** [comment]
Fixed the badly positioned (rotated) buttons in the cockpit in ffeb0ba1

**Comment by dirteat** [comment]
Thanks! And good job for the roll issue, it is gone too, perfectly smooth at all speed now!

**Comment by AnirudhP22** [comment]
Hi Richard,

I really like the amount of work that you have done for F-14 in FlightGear.
Previously I was using old version of FG and F-14b was working well, but now I am using FlightGear v 2020.3.3 & aircraft is not loading.
So I trayed to use  F-14 model V1.3 (FG 3.6+) from link = http://www.zaretto.com/sites/zaretto.com/files/F-14-13-fg36-2015-11-30.zip
Looks like its not compatible with FG v 2020.3.3 so I also tried to Clone Develop Repository (https://github.com/Zaretto/fg-aircraft/tree/develop) but unable to Clone it, I am getting error.
So i also tried to Download .Zip from Github.
I managed to download 378MB file, but while extracting .Zip file i ma getting error as seen in attached screenshot below.
![Error](https://user-images.githubusercontent.com/72618308/119372654-2c2af200-bcd5-11eb-9342-23a042813764.jpg)
